test/db/cmd/cmd_fnj

```diff
@@ -2,8 +2,8 @@ NAME=fnj shows demangled symbols
 FILE=bins/elf/demangle-test-cpp
 CMDS=<<EOF
 aaa
-fj~{268}
-fnj~{268}
+fj~{269}
```

Comment:
do you know what is the additional flag it finds and why?

Reply:
Just other order.

Reply:
```diff
--- a1.txt	2020-05-10 13:48:04.756031447 +0300
+++ a2.txt	2020-05-10 13:51:17.398666647 +0300
@@ -39,7 +39,12 @@
 0x00000000 1 class.int__std::uninitialized_copy_std
 0x00000000 1 class.std::allocator_traits_std::allocator_int
 0x00000000 1 class.int___std
+0x00000000 8 rax
 0x00000000 8 rbx
+0x00000000 8 rcx
+0x00000000 8 rdx
+0x00000000 8 rsi
+0x00000000 8 rdi
 0x00000000 8 r8
 0x00000000 8 r9
 0x00000000 8 r10
@@ -48,12 +53,8 @@
 0x00000000 8 r13
 0x00000000 8 r14
 0x00000000 8 r15
-0x00000000 8 rdx
-0x00000000 8 rsi
-0x00000000 8 rcx
-0x00000000 8 rax
+0x00000000 8 rflags
 0x00000040 616 segment.PHDR
-0x00000044 8 rflags
 0x000002a8 28 section..interp
 0x000002a8 28 segment.INTERP
 0x000002a8 0 .interp
@@ -286,7 +287,6 @@
 0x00004080 0 _edata
 0x00004080 0 __TMC_END__
 0x00004080 0 __bss_start
-0x00004080 8 rdi
 0x00004088 0 _end
 0x00178000 8 rbp
 0x00178000 8 rsp
```

Reply:
Well `rdi` and `rflags` offsets are zero now.